Source/FieldSolver/WarpXPushFieldsHybridPIC.cpp

```diff
@@ -85,8 +91,21 @@ void WarpX::HybridPICEvolveFields ()
     // for the hybrid-PIC solver since current values are interpolated to
     // a nodal grid
     for (int lev = 0; lev <= finest_level; ++lev) {
+        ablastr::utils::communication::FillBoundary(
+            *m_fields.get(FieldType::rho_fp, lev),
+            m_fields.get(FieldType::rho_fp, lev)->nGrowVect(),
+            WarpX::do_single_precision_comms,
+            Geom(lev).periodicity(),
+            true
+        );
         for (int idim = 0; idim < 3; ++idim) {
-            m_fields.get(FieldType::current_fp, Direction{idim}, lev)->FillBoundary(Geom(lev).periodicity());
+            ablastr::utils::communication::FillBoundary(
```

Comment:
What is the reason for this change?

Reply:
same as above for rho.

Reply:
I guess this question was more about whether there is a specific reason to call `ablastr::utils::communication::FillBoundary(...)` rather than using `mf->FillBoundary(...)` as is currently done?

Reply:
I guess I was transitioning over to using the WarpX wrappers to make sure that the "single precision comms" is respected and the wrapper has the flag for doing the nodal sync. I am mainly trying to make the boundary fills be done in the same way for each MF.